charts/oes/values.yaml

```diff
@@ -281,6 +281,22 @@ sapor:
       spinAdminUsername: admin
```

Comment:
We should remove these as they are moved under ldap now.

Reply:
Ok